pkg/cmd/server/start.go

```diff
@@ -132,6 +129,19 @@ func NewCommandStartServer(etcdPath string, out, errOut io.Writer, builders []*b
 		"Setup delegated auth")
 	//flags.StringVar(&o.Kubeconfig, "kubeconfig", "", "Kubeconfig of apiserver to talk to.")
 	o.RecommendedOptions.AddFlags(flags)
+
+	klogFlags := flag.NewFlagSet("klog", flag.ExitOnError)
+	klog.InitFlags(klogFlags)
+	flags.AddGoFlagSet(klogFlags)
+
+	// Sync the glog and klog flags.
+	klogFlags.VisitAll(func(f *flag.Flag) {
+		goFlag := flag.CommandLine.Lookup(f.Name)
+		if goFlag != nil {
+			goFlag.Value.Set(f.Value.String())
```

Comment:
and klog flags are sync'd to golang flags in order to fix the compatibility issue